troposphere/views/app.py

```diff
@@ -224,18 +225,25 @@ def forbidden(request):
     user, but was found to be unauthorized to use Atmosphere by OAuth.
     Returns HTTP status code 403 Forbidden
     """
+    metadata = get_site_metadata()
     template_params = {}
 
     template_params["THEME_URL"] = "/themes/%s" % settings.THEME_NAME
     template_params['ORG_NAME'] = settings.ORG_NAME
     template_params['SITE_TITLE'] = settings.SITE_TITLE
     template_params['SITE_FOOTER'] = settings.SITE_FOOTER
+    template_params['USER_PORTAL_LINK'] = metadata.user_portal_link
+    template_params['USER_PORTAL_LINK_TEXT'] = metadata.user_portal_link_text
+    template_params['ACCOUNT_INSTRUCTIONS_LINK'] = \
+        metadata.account_instructions_link
+
     if hasattr(settings, "BASE_URL"):
         template_params['BASE_URL'] = settings.BASE_URL
 
     # If banner message in query params, pass it into the template
     if "banner" in request.GET:
         template_params['banner'] = request.GET['banner']
+
     response = render_to_response(
         'no_user.html',
```

Comment:
I guess the thing with having this in SiteMetadata right now is that whatever you put in the textbox is what you _expect_ to be used.